cfgov/processors/processors_common.py

```diff
@@ -0,0 +1,32 @@
+from BeautifulSoup import BeautifulSoup
```

Comment:
Thanks, kurt-- I swear there were a few days where we had both versions in our requirements.